variants/FYSETC_S6/variant.h

```diff
@@ -0,0 +1,182 @@
+#define PD13  61 //D61
+#define PD14  62 //D62
+#define PD15  63 //D63
+#define PE0   64 //D64
+#define PE1   65 //D65
+#define PE2   66 //D66
+#define PE3   67 //D67
+#define PE4   68 //D68
+#define PE5   69 //D69
+#define PE6   70 //D70
+#define PE7   71 //D71
+#define PE8   72 //D72
+#define PE9   73 //D73
+#define PE10  74 //D74
+#define PE11  75 //D75
+#define PE12  76 //D76
+#define PE13  77 //D77
+#define PE14  78 //D78
+#define PE15  79 //D79
+
+// This must be a literal with the same value as PEND
+#define NUM_DIGITAL_PINS        87
+// This must be a literal with a value less than or equal to to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       7
+#define NUM_ANALOG_FIRST        80
+
+// PWM resolution
+#define PWM_RESOLUTION          12
+#define PWM_FREQUENCY           20000 // >= 20 Khz => inaudible noise for fans
+#define PWM_MAX_DUTY_CYCLE      255
+
+// SPI Definitions
+#define PIN_SPI_SS              PA4
+#define PIN_SPI_MOSI            PA7
+#define PIN_SPI_MISO            PA6
+#define PIN_SPI_SCK             PA5
+
+// I2C Definitions
+#define PIN_WIRE_SDA            PB9
+#define PIN_WIRE_SCL            PB8
+
+// Timer Definitions
+// Do not use timer used by PWM pin. See PinMap_PWM.
+#define TIMER_TONE              TIM6
+#define TIMER_SERVO             TIM2
+#define TIMER_SERIAL            TIM7
+
+// UART Definitions
+//#define SERIAL_UART_INSTANCE    1 // Connected to EXP3 header
+/* Enable Serial 3 */
+#define HAVE_HWSERIAL1
+#define HAVE_HWSERIAL3
+
+// Default pin used for 'Serial' instance (ex: ST-Link)
+// Mandatory for Firmata
+#define PIN_SERIAL_RX           PA10
+#define PIN_SERIAL_TX           PA9
+
+/* HAL configuration */
+#define HSE_VALUE               12000000U
+
+#define FLASH_PAGE_SIZE         (4U * 1024U)
+
+#ifdef __cplusplus
+} // extern "C"
+#endif
+
+/*----------------------------------------------------------------------------
+ *        Arduino objects - C++ only
+ *----------------------------------------------------------------------------*/
+
+#ifdef __cplusplus
+// These serial port names are intended to allow libraries and architecture-neutral
+// sketches to automatically default to the correct port name for a particular type
+// of use.  For example, a GPS module would normally connect to SERIAL_PORT_HARDWARE_OPEN,
+// the first hardware serial port whose RX/TX pins are not dedicated to another use.
+//
+// SERIAL_PORT_MONITOR        Port which normally prints to the Arduino Serial Monitor
+//
+// SERIAL_PORT_USBVIRTUAL     Port which is USB virtual serial
+//
+// SERIAL_PORT_LINUXBRIDGE    Port which connects to a Linux system via Bridge library
+//
+// SERIAL_PORT_HARDWARE       Hardware serial port, physical RX & TX pins.
+//
+// SERIAL_PORT_HARDWARE_OPEN  Hardware serial ports which are open for use.  Their RX & TX
+//                            pins are NOT connected to anything by default.
+#define SERIAL_PORT_MONITOR     Serial
+#define SERIAL_PORT_HARDWARE_OPEN  Serial
```

Comment:
If you didn't define Serial then specify which one you want use: Serial1 or Serial3.

Reply:
The default serial i want is SerialUSB.

Reply:
Yes this would be best and inline with your though.

Reply:
Sorry, leave
`#define SERIAL_PORT_MONITOR Serial`
and add 
`#define SERIAL_PORT_USBVIRTUAL SerialUSB`

As I wrote when USB is enable then it will be the default `Serial`